README.md

```diff
@@ -8,6 +8,9 @@
 # Impersonate Component
 A component that stores the current authentication session and creates new session for impersonating Users. User can revert back to original authentication sessions without the need to re-login.
 
+### Warning
+Always double check that you cannot "spoof" other users in the controller actions. To prevent hijacking of users accounts that the current authenticated User shouldn't/wouldn't have normal access to. This Plugin does circumvent default authentication mechanisms.
```

Comment:
not "you", but an attacker
if you as admin want that (and it is in the guidelines of the website etc), thats fine
but an introduce shouldnt be able to reverse engineer a way to become a user or even admin.
just saying :)